articles/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker.md

```diff
@@ -59,6 +59,9 @@ Run the following commands on all **iSCSI target virtual machines**.
    <pre><code>sudo zypper update
    </code></pre>
 
+> [!NOTE]
+   > OS reboot may be needed after the OS upgrade has been completed successfully. 

```

Comment:
I think the spaces at the beginning have to be removed

Reply:
Fixed